src/linpde_gp/randprocs/covfuncs/_independent_multi_output.py

```diff
@@ -62,8 +62,8 @@ def _evaluate_jax(self, x0: jnp.ndarray, x1: Optional[jnp.ndarray]) -> jnp.ndarr
 
         return result
 
-    def _evaluate_linop(
-        self, x0: np.ndarray, x1: Optional[np.ndarray]
+    def linop(
+        self, x0: pn.utils.ArrayLike, x1: Optional[pn.utils.ArrayLike] = None
```

Comment:
```suggestion
        self, x0: ArrayLike, x1: Optional[ArrayLike] = None
```

and then `from probnum.typing import ArrayLike` at the top of the file